h/browser/chrome/karma.config.js

```diff
@@ -32,6 +41,9 @@ module.exports = function(config) {
       '../../static/scripts/blocklist.js'
     ],
 
+    proxies: {
+      "/settings.json": "http://localhost:9877/base/test/settings.json"
```

Comment:
I've changed this to a relative URL, but left the port change in. Even with a relative URL it still uses the port from the Karma config file, not the port that Karma is actually running on, to fetch the file from.